### PR TITLE
Adding Amperity Quickstart

### PR DESCRIPTION
**Pull request by djperlovsky**
Adding quickstart + assets for Building a Unified Customer 360 using Snowflake + Amperity. Followed all instructions in [the guide](https://github.com/Snowflake-Labs/sfquickstarts?tab=readme-ov-file#snowflake-quickstart-guides)

I validated the changes locally -> only note is that the assets I added didn't show up in localhost. I assume that is an environment thing. All links work + styling looks correct. 
image

Per the instructions, I accept the Contributor License Agreement

Let me know if you have any questions or concerns

### CODE REVIEW

File: site/sfguides/src/build-unified-customer-360-using-amperity/build-unified-customer-360-using-amperity.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
+author: Daniel Perlovsky
+id: build-unified-customer-360-using-amperity
+summary: In this quickstart we will walk through the basic steps to turn raw data into a world class Customer 360 collection of tables in hours instead of months using Amperity.
+categories: Getting-Started, Marketing, Data-Engineering
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Twitter
```
Comment (iamontheinet) [inline site/sfguides/src/build-unified-customer-360-using-amperity/build-unified-customer-360-using-amperity.md]: @djperlovsky Remove tags that don't apply to this QS.

File: site/sfguides/src/build-unified-customer-360-using-amperity/build-unified-customer-360-using-amperity.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
+author: Daniel Perlovsky
+id: build-unified-customer-360-using-amperity
+summary: In this quickstart we will walk through the basic steps to turn raw data into a world class Customer 360 collection of tables in hours instead of months using Amperity.
+categories: Getting-Started, Marketing, Data-Engineering
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Twitter 
+
+# Build a Unified Customer 360 Database using Amperity
```
Comment (iamontheinet) [inline site/sfguides/src/build-unified-customer-360-using-amperity/build-unified-customer-360-using-amperity.md]: @djperlovsky Add "and Snowflake" at the end of the title.

File: site/sfguides/src/build-unified-customer-360-using-amperity/build-unified-customer-360-using-amperity.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+![CTSU](assets/ctsu.png)
+
+In this quickstart we will walk through the basic steps to turn raw data into a world class Customer 360 collection of tables in hours instead of months.
+
+![Medallion architecture](assets/medallion-architecture.png)
+
+### Prerequisites
+
+- Access to a Snowflake account with the ability to set up secure data sharing
```
Comment (iamontheinet) [inline site/sfguides/src/build-unified-customer-360-using-amperity/build-unified-customer-360-using-amperity.md]: @djperlovsky Add link to Snowflake signin/signup page.

**Review by iamontheinet — CHANGES_REQUESTED** [review]

**Comment by djperlovsky** [comment]
@iamontheinet Responded to the feedback and uploaded new changes. Any other feedback or am I good to merge?

**Comment by djperlovsky** [comment]
@iamontheinet @sfc-gh-lambrosetti Can I get some eyes here?

**Comment by sfc-gh-lambrosetti** [comment]
@iamontheinet - this looks good to me. images didn't load locally because of the typical issue there, but otherwise, everything else looks great!

**Pull request merged**